index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+
+<html lang="en" dir="ltr">
+<head>
+	<meta charset="utf-8">
+	<link rel="stylesheet" href="stylesheets/normalize.css">
+	<link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+	<link rel="stylesheet" href="stylesheets/styles.css">
+	<title>michelle cronin.</title>
+</head>
+<body>
+	<div class="site-container">
+
+		<header>
+			<div class="title">
```

Comment:
This file uses tabs for indentation, but the rest use spaces. What's up?

Reply:
not sure. i tend to use tab for indentation. maybe command+i??